src/common/components/Popover/Popover.theme.ts

```diff
@@ -5,16 +5,19 @@ export type PopoverVariants = VariantProps<typeof popoverTheme>;
 export const popoverTheme = tv(
   {
     base: [
-      "bg-bg-base",
-      "z-50",
-      "w-72",
-      "rounded-md",
+      "bg-surface-base",
+      "z-popover",
+      "w-screen", // Required to make popover full width on mobile
+      "max-w-[320px]",
+      "min-w-0",
+      "rounded-xl",
+      "py-5",
+      "px-5",
       "border",
-      "p-4",
-      "shadow-md",
+      "border-border-base",

```

Comment:
issue:
don't think `border-border-base` exists, did you mean [`border-border-default`](https://github.com/AfterClass-io/afterclass.io-v2/blob/60-featui-popover-and-command-styling/src/common/tools/tailwind/themes/appTheme.ts#L32-L37)?

```suggestion
      "border-border-default",
```

![image](https://github.com/AfterClass-io/afterclass.io-v2/assets/13061926/677b14c2-37ce-48a9-9c6c-5b1dfb94d453)

Reply:
Oh right. Nice catch. Fixed it here https://github.com/AfterClass-io/afterclass.io-v2/pull/62/commits/38926eb13ce03403f889b6c8016f3f270fc83695